recipes-openxt/xenclient/xenclient-dom0-tweaks/openxt.cfg

```diff
@@ -1,37 +1,35 @@
 [global]
 default=openxt-normal
+tboot=tboot min_ram=0x2000000 loglvl=all serial=115200,8n1,0x3f8 logging=serial,memory
+sinit=GM45_GS45_PM45_SINIT_51.BIN Q35_SINIT_51.BIN Q45_Q43_SINIT_51.BIN i5_i7_DUAL_SINIT_51.BIN i7_QUAD_SINIT_51.BIN 3rd_gen_i5_i7_SINIT_67.BIN Xeon-5600-3500-SINIT-v1.1.bin Xeon-E7-8800-4800-2800-SINIT-v1.1.bin 4th_gen_i5_i7_SINIT_75.BIN 5th_gen_i5_i7_SINIT_79.BIN 6th_gen_i5_i7_SINIT_71.BIN 7th_gen_i5_i7-SINIT_74.bin 8th_gen_i5_i7-SINIT_76.bin
+ucode=microcode_intel.bin
 
 [openxt-normal]
-options=console=com1 dom0_mem=min:420M,max:420M,420M efi=no-rs,attr=uc com1=115200,8n1,pci mbi-video vga=current flask=enforcing loglvl=debug guest_loglvl=debug
+options=placeholder console=com1 dom0_mem=min:420M,max:420M,420M efi=no-rs,attr=uc com1=115200,8n1,pci mbi-video vga=current flask=enforcing loglvl=debug guest_loglvl=debug
```

Comment:
Shouldn't placeholder be added to all the entries of openxt.cfg?

Reply:
No, it's only needed for xen.efi

Reply:
I was referring to xen.efi for all the openxt-support-* entries in openxt.cfg.  But unsealing doesn't work for those, so it would only be for consistency.